Python/bytecodes.c

```diff
@@ -2128,7 +2128,18 @@ dummy_func(
         op(_GUARD_TYPE_VERSION, (type_version/2, owner -- owner)) {
             PyTypeObject *tp = Py_TYPE(PyStackRef_AsPyObjectBorrow(owner));
             assert(type_version != 0);
-            EXIT_IF(tp->tp_version_tag != type_version);
+            EXIT_IF(FT_ATOMIC_LOAD_UINT32_RELAXED(tp->tp_version_tag) != type_version);
```

Comment:
The accesses to `tp_version_tag` should use `FT_ATOMIC_LOAD_UINT_RELAXED` instead of `FT_ATOMIC_LOAD_UINT32_RELAXED` because it's an `unsigned int`.

Reply:
Oh... we have a lot of these that should be changed. We can do in a separate PR.